docs/source/workflow_inputs.rst

```diff
@@ -79,6 +79,17 @@ The OpenStudio-ERI calculation(s) to be performed are entered in ``/HPXML/Softwa
   For example, it does not check that the home meets all ENERGY STAR Mandatory Requirements.
   It is the software tool's responsibility to perform these additional steps. 
 
+The version of the Zero Energy Ready Home calculation is entered in ``/HPXML/SoftwareInfo/extension/ZERHCalculation``.
+
+  ===========  ========  =======  ===========  ========  =======  =========================================
+  Element      Type      Units    Constraints  Required  Default  Description
+  ===========  ========  =======  ===========  ========  =======  =========================================
+  ``Version``  string             See [#]_     No [#]_            Version of Zero Energy Ready Home program
+  ===========  ========  =======  ===========  ========  =======  =========================================
+  
+  .. [#] Version choice is "Version_1".
+  .. [#] Version only required to run Zero Energy Ready Home calculation.
+
```

Comment:
Can you incorporate this as another row into the existing table above (w/ `EnergyStarCalculation/Version`)? Also, I suggest generalizing the language of the ENERGY STAR ERI warning right above this to accommodate ZERH.